src/app/cmdoptions.cpp

```diff
@@ -319,12 +319,16 @@ namespace
     }
 
     constexpr const BoolOption SHOW_HELP_OPTION {"help", 'h'};
+#if !defined(Q_OS_WIN) || defined(DISABLE_GUI)
```

Comment:
It doesn't look expected to me.
In addition, I would refrain from unrelated changes that may slow down the review of main PR changes.